frontend/src/components/Graph/index.js

```diff
@@ -175,7 +179,7 @@ export class Graph extends React.Component {
 		});
 		
 		// create accrued state lines
-		let lastTime = 0;
+		let lastTime = data[0].time;
```

Comment:
Note that here I assume that the first item in the array returned by the `/events` endpoint is always the least recent date (e.g. IED?), and there is at least one item in the array. This is not necessary to fix the margins, just cosmetics. Setting `lastTime = 0;` starts the line somewhere on the far left, which is not what you want, I suppose.